src/content/docs/devtools/Docker/docker-swarm-case-study.md

```diff
@@ -0,0 +1,206 @@
+| Attribute  | Details                                                      |
```

Comment:
Please double check the formatting for markdown `frontmatter`. Here is an example:

```md
---
title: TensorFlow
author: Vegard Lekve <vegardlekve>
tags: Machine learning, Neural networks, JavaScript, Python
---
```